site/content/docs/0.0/getting-started/contribute.md

```diff
@@ -10,4 +10,61 @@ aliases:
   - "/docs/0.0/getting-started/build-tools/"
 ---
 
-{{< callout-soon "page" >}}
+## Tooling setup
+
+OUDS Web uses [npm scripts](https://docs.npmjs.com/misc/scripts/) to build the documentation and compile source files. Our [package.json]({{< param repo >}}/blob/v{{< param current_version >}}-ouds-web/package.json) houses these scripts for compiling code, running tests, and more. These aren't intended for use outside our repository and documentation.
```

Comment:
[BS side] Same on this page (but not all) as well since we are cloning main most of the time, check all the files before submitting a PR down there.